src/zeek-setup.cc

```diff
@@ -556,9 +556,13 @@ SetupResult setup(int argc, char** argv, Options* zopts) {
         reporter->FatalError("-u incompatible with --no-unused-warnings");
 
 #ifdef DEBUG
-    if ( options.debug_log_streams ) {
+    if ( options.debug_log_streams )
         debug_logger.EnableStreams(options.debug_log_streams->data());
 
+    if ( getenv("ZEEK_DEBUG_LOG_STREAMS") )
+        debug_logger.EnableStreams(getenv("ZEEK_DEBUG_LOG_STREAMS"));
```

Comment:
Nit: can reuse previous the result:

```suggestion
    if ( auto streams = getenv("ZEEK_DEBUG_LOG_STREAMS") )
        debug_logger.EnableStreams(streams);
```